vmdb/app/models/orchestration_stack_openstack.rb

```diff
@@ -1,12 +1,16 @@
 class OrchestrationStackOpenstack < OrchestrationStack
   def raw_update_stack(options)
-    ext_management_system.with_provider_connection(:service => "Orchestration") do |connection|
+    options = {:service => "Orchestration"}
```

Comment:
Actually, this will break the update itself because the options for the save call below are being lost.